sections/image-banner.liquid

```diff
@@ -35,6 +35,11 @@
     assign full_width = '120vw'
     assign stacked_sizes = '(min-width: 750px) 60vw, 120vw'
     assign widths = '450, 660, 900, 1320, 1800, 2136, 2400, 3600, 7680'
+  elsif section.settings.image_behavior == 'fixed'
+    assign half_width = '100vw'
+    assign full_width = '100vw'
+    assign stacked_sizes = '(min-width: 750px) 100vw'
```

Comment:
Actually I don't think this is quite right 🤔 
I don't even think we need this specific use case. Everything in the next `else` is what should be applied. 

- `stacked_sizes` is applied when we have two images and it's set to be stacked on mobile. So it should be `50vw` on viewport greater than `750px` and `100vw` on mobile. 

- `half_width` is applied when we detect two images added and it's not using a stacked layout on mobile. So `50vw` should be used. 
- `full_width` is used in any other case which seem right as well.

Reply:
@ludoboludo 

Actually, that is not the case for fixed position background, because the background image will not scale down to match the container width. It just crops. This type of image will always fill 100% of the viewport width. Try changing the width of the container - the image won't scale to fit, it just reduces the cropped area.

We must serve `100vw` in all scenarios to avoid serving a low-res image (see the screenshots I shared above with [the before & after](https://github.com/Shopify/dawn/pull/2524#discussion_r1176598698))